doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -132,7 +132,7 @@ Update ACRN hypervisor Image
    .. note::
 
       Double check PCI devices BDF defined in the
-      ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``
+      ``hypervisor/arch/x86/configs/nuc7i7dnb/pci_devices.h``
```

Comment:
Do you also need to update ``hardware.rst`` to complete this change to supported hardware?  
@NanlinXie note this supported HW change should be mentioned in the release notes.

Reply:
Yes, we will have another separate patch for hardware.rst. Let's merge this patch firstly.

Reply:
Ah, OK.  You're switching from using the kit identification (nuc7i7bnh) to the board identification (nuc7i7dnb), so it's not really a change in supported hardware.